spec.bs

```diff
@@ -395,14 +395,12 @@ To <dfn>append private state token issue request headers</dfn> given a [=/reques
 
  1. Let |tokens| be the result of [=generate masked tokens|generating masked tokens=] with |numTokens|.
  1. Configure the HTTP request.
-               a. Set a load flag to bypass the HTTP cache.
-          b. Add <a http-header>Sec-Private-State-Token</a> request header containing a
-                      base64-encoded version of the bytestring of tokens
+               1. Set a load flag to bypass the HTTP cache.
+               1. Add <a http-header>Sec-Private-State-Token</a> [=header list/set a structured field value|structured request header=]
+                      containing a base64-encoded version of the bytestring of tokens
                       generated in Step 11.
-               c. Add <a http-header>Sec-Private-State-Token-Version</a> request header that specifies
-                      the version of the cryptographic protocol used.
-
-     Issue: This step should append structured headers using [=header list/set a structured field value=]
+               1. Add <a http-header>Sec-Private-State-Token-Version</a> [=header list/set a structured field value|structured request header=]
```

Comment:
See comment above, and we should also follow up here with how to get the version.